source/tutorials/longitudinal.rst

```diff
@@ -115,6 +115,50 @@ Second, control charts display the mean value of "metric" at each "state". The f
 This visualizer currently only generates control charts, which are a useful **qualitative** approach for visually identifying abnormal time points; accompanying statistical tests may be added in future releases.
 
 
+First differencing to track rate of change
+------------------------------------------
+Another way to view time series data is by assessing how the rate of change differs over time. We can do this through calculating first differences, which is the magnitude of change between successive time points. If :math:`Y_\text{t}` is the value of metric :math:`Y` at time :math:`t`, the first difference at time :math:`t`, :math:`{\Delta}Y_\text{t} = Y_\text{t} - Y_\text{t-1}`. This transformation is performed in the ``first-differences`` method in ``q2-longitudinal``.
+
+.. command-block::
+
+   qiime longitudinal first-differences \
+     --m-metadata-file ecam-sample-metadata.tsv \
+     --m-metadata-file shannon.qza \
+     --p-state-column month \
+     --p-metric shannon \
+     --o-first-differences shannon-first-differences \
+     --p-individual-id-column studyid \
+     --p-replicate-handling random
+
+This outputs a ``SampleData[FirstDifferences]`` artifact, which can then be viewed, e.g., with the ``volatility`` visualizer or analyzed with ``linear-mixed-effects`` or other methods.
+
+A similar method is ``first-distances``, which instead identifies the beta diversity distances between successive samples from the same subject. The pairwise distance between all samples can already be calculated by the ``beta`` or ``core-metrics`` methods in ``q2-diversity``, so this method simply identifies the distances between successive samples collected from the same subject and outputs this series of values as metadata that can be consumed by other methods.
+
+.. command-block::
+
+   qiime longitudinal first-distances \
```

Comment:
I'm getting the following error when running this command:

```
$ qiime longitudinal first-distances \
>   --i-distance-matrix unweighted_unifrac_distance_matrix.qza \
>   --m-metadata-file ecam-sample-metadata.tsv \
>   --p-state-column month \
>   --o-first-distances first-distances \
>   --p-individual-id-column studyid \
>   --p-replicate-handling random


Plugin error from longitudinal:

  Missing samples in metadata: {'L1S208', 'L5S240', 'L2S155', 'L4S137', 'L2S309', 'L1S140', 'L1S57', 'L2S222', 'L2S357', 'L5S104', 'L2S240', 'L1S281', 'L3S378', 'L4S63', 'L1S257', 'L3S341', 'L6S20', 'L1S8', 'L3S313', 'L6S68', 'L4S112', 'L5S155', 'L3S360', 'L5S174', 'L2S204', 'L5S203', 'L3S294', 'L1S76', 'L6S93', 'L2S175', 'L1S105', 'L2S382', 'L5S222'}

Debug info has been saved to /tmp/qiime2-q2cli-err-ijndejsh.log
```

Reply:
sorry about that @jairideout ! I think the metadata file I'm using locally and the one in `data.qiime2.org` got mixed up.